src/features/channels/components/channel-list/search-channel/SearchChannel.tsx

```diff
@@ -97,20 +105,21 @@ const ChannelList = () => {
                       <Box key={channel.channelName}>
```

Comment:
I suggest putting the whole 'Modal' component into its own component, maybe call it 'HiddenChannelsModal' so you don't have as much code in a single file

Reply:
Yes, I was wondering about that too.

Reply:
It's done and you will see it in the next commit.